glusterd2/events/eventhandler.go

```diff
@@ -154,3 +162,66 @@ func (h *handler) Handle(e *api.Event) {
 func (h *handler) Events() []string {
 	return h.events
 }
+
+func getJWTToken(eventtype, secret string) string {
+	claims := &jwt.StandardClaims{
+		IssuedAt:  time.Now().Unix(),
+		ExpiresAt: time.Now().Add(time.Second * 10).Unix(),
+		Subject:   eventtype,
+	}
+
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	ss, err := token.SignedString([]byte(secret))
+	if err != nil {
+		return ""
+	}
+	return ss
+}
+
+//WebhookPublish sends event to registered webhooks
+func WebhookPublish(webhook *eventsapi.Webhook, e *api.Event) error {
+	message, err := json.Marshal(e)
+	if err != nil {
+		log.WithError(err).Error("failed to marshal event")
```

Comment:
We can also log the event here, just the event name. log with field.

Reply:
done